cockatrice/src/customlineedit.h

```diff
@@ -0,0 +1,24 @@
+
+#ifndef CUSTOMLINEEDIT_H
+#define CUSTOMLINEEDIT_H
+
+#include <QLineEdit>
+
+class QKeyEvent;
+class QWidget;
+class QString;
+
+// Should be used when the there is a risk of conflict between line editor
+// shortcuts and other shortcuts
+class CustomLineEdit : public QLineEdit
```

Comment:
CustomLineEdit is to vague in my opinion, this is specifically a lineedit that you can unfocus with a key.